test/index.js

```diff
@@ -682,9 +682,10 @@ describe('hpal', () => {
 
                 flags.onCleanup = async () => await rimraf('new/my-project');
 
-                const cli = RunUtil.cli(['new', 'my-project'], 'new');
+                const projectName = 'my-project';
+                const cli = RunUtil.cli(['new', projectName], 'new');
 
-                answerNpmInit(cli, 'chosen-name');
+                answerNpmInit(cli, projectName);
```

Comment:
By making the project name and chosen name identical, it's a little bit hard to see that this test is preferring the chosen name.  Can we change it back to the way it was, but keep the new assertion below?  If you'd rather not I am happy to pick this up!

Reply:
ohhhh I gotcha. Yea, that makes sense, I'll fix that up. Thanks for catching that!